pages/lists/[listId]/activity.tsx

```diff
@@ -80,14 +83,14 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
         topContributor: mostActiveData?.data?.length ? mostActiveData.data[0] : null,
         projectData: projectData ?? [],
       },
+      isOwner: list && list.user_id === userId,
```

Comment:
This is the same logic used on the Overview page.

Reply:
Thanks, @nickytonline !!!